tests/execution_engine/test_sqlalchemy_execution_engine.py

```diff
@@ -173,66 +171,6 @@ def test_instantiation_via_fluent_data_sources_with_kwargs(
     assert execution_engine.engine
 
 
-@pytest.mark.trino
-def test_instantiation_via_fluent_data_source__trino_add_sql(sa, empty_data_context):
```

Comment:
I think we can leverage the Trino test in `tests/datasource/fluent/integration/test_sql_datasources.py` instead of this